egs/lrs/avsr1/path.sh

```diff
@@ -0,0 +1,38 @@
+# If project is in ESPnet directory (e.g. espnet/egs/project/avsr1), the MAIN_ROOT path can be set with:
+MAIN_ROOT=$PWD/../../.. 
+KALDI_ROOT=$MAIN_ROOT/tools/kaldi
+ESPNET_VENV="espnet_venv"
+
+# Setting path variables for dataset, OpenFace, DeepXi, pretrained model and musan
+# Change this variables and adapt it to your Folder structure
+export DATA_DIR="/home/foo/LRS2"					# The LRS2 dataset directory
```

Comment:
I think it is better to add the installation of these tools in https://github.com/espnet/espnet/tree/master/tools/installers